src/internal/internal_hettmqr.cc

```diff
@@ -71,7 +71,7 @@ void hettmqr(
     Matrix<scalar_t>& V,
     Matrix<scalar_t>& T,
     HermitianMatrix<scalar_t>& C,
-    int tag, Options const& opts )
+    int tag_base, Options const& opts )
```

Comment:
Maybe add a description of it somewhere. I guess there is a range or something to take into account.